homeassistant/components/bluetooth/config_flow.py

```diff
@@ -112,3 +121,39 @@ async def async_step_user(
     ) -> FlowResult:
         """Handle a flow initialized by the user."""
         return await self.async_step_multiple_adapters()
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: ConfigEntry,
+    ) -> OptionsFlowHandler:
+        """Get the options flow for this handler."""
+        return OptionsFlowHandler(config_entry)
+
+
+class OptionsFlowHandler(OptionsFlow):
+    """Handle the option flow for bluetooth."""
+
+    def __init__(self, config_entry: ConfigEntry) -> None:
+        """Initialize options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle options flow."""
+        if user_input is not None:
+            return self.async_create_entry(title="", data=user_input)
+
+        if platform.system() != "Linux":
+            return self.async_abort(reason="no_options")
```

Comment:
There is a callback you can implement to indicate no options flow

Reply:
https://github.com/home-assistant/core/blob/2497ff5a39292b7167733206fe2e1e63a6a3b86e/homeassistant/config_entries.py#L1275-L1278

Reply:
Thanks